interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
Also note in turns of mapping, could consider the option_id meaningless(ie. simply used to place the entry in the list_options table) rather than try to define it for use. For example, if the add_edit_issue.php script had 2 forms, then could just use:
add_edit_issue_1
add_edit_issue_2
for the option_id
Which again would be meaningless for mapping, but important for placing them in list_options and having an anchor to change the rules (for example, if needed to change add_edit_issue_2 to fix a bug in the sql upgrade script)